### PR TITLE
Fix warnings

### PR DESCRIPTION
**Pull request by asmfstatoil**

### CODE REVIEW

File: src/main/java/neqsim/physicalProperties/physicalPropertyMethods/liquidPhysicalProperties/density/Costald.java [hunk 1]
```diff
@@ -112,10 +111,12 @@ public double calcDensity() {
                 * liquidPhase.getPhase().getComponents()[i].getCriticalVolume())
             * liquidPhase.getPhase().getComponents()[i].getx();
 
-        tempVar += liquidPhase.getPhase().getComponents()[i].getx()
-            * (liquidPhase.getPhase().getComponents()[i].getVolumeCorrection()
-                + liquidPhase.getPhase().getComponents()[i].getVolumeCorrectionT()
-                    * (liquidPhase.getPhase().getTemperature() - 288.15));
+        /*
```
Comment (asmfstatoil) [inline src/main/java/neqsim/physicalProperties/physicalPropertyMethods/liquidPhysicalProperties/density/Costald.java:114]: Unused variable

File: src/main/java/neqsim/processSimulation/processEquipment/heatExchanger/HeatExchanger.java [hunk 2]
```diff
@@ -270,10 +270,10 @@ public void run(UUID id) {
       outStream[0].run(id);
       run(id);
     } else {
-      double cP0 = inStream[0].getThermoSystem().getCp();
-      double cP1 = inStream[1].getThermoSystem().getCp();
       int streamToCalculate = 0;
 
+      // double cP0 = inStream[0].getThermoSystem().getCp();
```
Comment (asmfstatoil) [inline src/main/java/neqsim/processSimulation/processEquipment/heatExchanger/HeatExchanger.java:275]: unused variables

File: src/main/java/neqsim/thermo/component/ComponentHydrateGF.java [hunk 3]
```diff
@@ -45,6 +45,7 @@ public ComponentHydrateGF(String component_name, double moles, double molesInPha
           dataSet.getString("ID");
         } catch (Exception ex) {
           logger.info("no parameters in tempcomp -- trying comp.. " + component_name);
+          dataSet.close();
```
Comment (asmfstatoil) [inline src/main/java/neqsim/thermo/component/ComponentHydrateGF.java:48]: must close dataSet before overwriting it.

File: src/test/java/neqsim/processSimulation/processEquipment/separator/SeparatorTest.java [hunk 4]
```diff
@@ -41,11 +41,10 @@ public void setUpBeforeClass() throws Exception {
     Separator sep = new Separator("inlet separator");
     sep.setInletStream(inletStream);
 
-    StreamInterface gasFromSep = new Stream("liquid from separator", sep.getLiquidOutStream());
-    StreamInterface liqFromSep = new Stream("liquid from separator", sep.getLiquidOutStream());
-
     processOps.add(inletStream);
     processOps.add(sep);
+    // StreamInterface gasFromSep = new Stream("liquid from separator", sep.getLiquidOutStream());
```
Comment (asmfstatoil) [inline src/test/java/neqsim/processSimulation/processEquipment/separator/SeparatorTest.java:46]: unused streams

**Review by EvenSol — APPROVED** [review]
good updates!

**Pull request merged**